src/User/src/Entity/User.php

```diff
@@ -32,21 +32,21 @@ class User extends AbstractEntity
 
     /**
      * @ORM\OneToOne(targetEntity="UserAvatar", cascade={"persist", "remove"}, mappedBy="user")
-     * @var UserAvatar $avatar
+     * @var UserAvatar|null $avatar
      */
-    protected $avatar;
+    protected ?UserAvatar $avatar;
```

Comment:
`POST /account/register` does create a new user account, but outputs the following message:
```
{
    "error": {
        "messages": [
            "Typed property Api\\User\\Entity\\User::$avatar must not be accessed before initialization"
        ]
    }
}
```

**Cause**:
When a new user account is registered, the avatar is not set so it when we call User->getArrayCopy(), we try to access the property `$avatar`, which must return either `null` or an instance of `UserAvatar`.

**Solution**:
In `src/User/src/Entity/User.php` initialize the property `$avatar` with `null`:
`protected ?UserAvatar $avatar = null;`

Reply:
Done.